### PR TITLE
some Pep8 changes

### PR DESCRIPTION
**Pull request by test-user-000**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/test-user-000/zfs-tests/pull/1" target="_blank">ReviewNinja

**Comment by test-user-000** [comment]
I think its p good :+1:

**Comment by test-user-000** [comment]
I think its bad !fix

File: CleanupRuns.py [hunk 1]
```diff
@@ -18,8 +18,7 @@
 
 subprocess.check_call(['zfs', 'destroy', '-r', runs_directory])
 
-subprocess.check_call(['zfs', 'create', 
-    Configs.test_filesystem_path + '/runs'])
+subprocess.check_call(['zfs', 'create',
+                      Configs.test_filesystem_path + '/runs'])
```
Comment (test-user-000) [inline CleanupRuns.py:22]: !fix I dont like the spacing

**Pull request merged**